### ISSUE TITLE
version.yaml syntax error

### ISSUE THREAD

**Issue by angelcoding** [OFFLINE-GmbH/oc-gdpr-plugin#73]
I'm getting an error when running `php artisan october:up` ...
```
A syntax error was detected in .../plugins/offline/gdpr/updates/version.yaml. Unable to parse at line 41 (near " fields to optimize ePrivacy confor
  mity. (*Very big thank you* to ayumihamsaki for giving a lot of feedback via our") at line 45 (near ".../vendor/october/rain/src/Parse/Yaml.php").
```
I think it's because the first comment for 1.1.1 is spanning multiple lines when it should only span a single line.
Hopefully you will fix?
Thanks.

**Comment by damsfx** [comment]
@tobias-kuendig 
Same error with `php artisan plugin:install Offline.Gdpr` #72   

Also getting the error form October's plugin store!

**Comment by tobias-kuendig** [comment]
Closed via #74
